src/definitions/behaviors/form.js

```diff
@@ -951,6 +951,7 @@
                         if (rule === undefined) {
                             module.debug('Removed all rules');
                             validation[field].rules = [];
+                            delete validation[field];
```

Comment:
This breaks into a console error in case one wants to remove rules and validate the field afterwards, because the `validate.field` method expects a field to be existent inside the validation array (even without any rule)
See https://jsfiddle.net/lubber/m51yfaek/7/ and click the purple button

![image](https://user-images.githubusercontent.com/18379884/209808841-527eff14-c7b6-4de9-bbbe-e978289e9cae.png)

Reply:
This error is already triggered without my modifications. In this case I only remove the rules if the field is removed. But you're right I should only delete the validation array key if the field doesn't exist anymore.

Reply:
You could just fix this inside the validate.field function to fetch any non existing validation array access.
And if you leave the "delete", you could also remove the previous `validation[field].rules = [];` because that wont be necessary anymore if you delete the whole field key from validatation array anyway

Reply:
Actually your fiddle seems broken since you try to validate a fied that doesn't exist: `field2` instead of `field-2`.

`validate.field` should indeed check if a field exists before trying to validate it.

Reply:
> Actually your fiddle seems broken since you try to validate a fied that doesn't exist: `field2` instead of `field-2`.
> 
> `validate.field` should indeed check if a field exists before trying to validate it.

Oh, that's indeed right!  However, i still dont understand from the code, why "field.identifier" does not break, although validation[field] was previously deleted and should be undefined .. I investigate